packages/cli-main/package.json

```diff
@@ -51,7 +51,7 @@
     ]
   },
   "dependencies": {
-    "@oclif/core": "1.16.4",
+    "@oclif/core": "1.9.2",
     "@oclif/plugin-commands": "^2.2.0",
     "@oclif/plugin-help": "^5.1.14",
```

Comment:
these two should also be nudged back:
<img width="299" alt="image" src="https://user-images.githubusercontent.com/234027/197573283-0f76d4a8-8a1a-4e9a-985f-1f653db2535a.png">

Reply:
@shauns excellent point, I've updated those as well.